charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         {{- if eq .Values.backend.settings "prod" }}
         args: ["-c", "uwsgi --http :{{ .Values.backend.service.port }} --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.prod "]
         {{- else }}
-        args: ["-c", "python manage.py runserver --noreload 0.0.0.0:{{ .Values.backend.service.port }} --settings=backend.settings.server.{{ .Values.backend.settings }}"]
+        args: ["-c", "unset http_proxy; unset https_proxy; python manage.py runserver --noreload 0.0.0.0:{{ .Values.backend.service.port }} --settings=backend.settings.server.{{ .Values.backend.settings }}"]
```

Comment:
I think this cannot work as `http_proxy` and `https_proxy` are not defined in our local setup but also on servers.
What is the reason of this `unset` of those env. variables ?

Reply:
Regarding this issue seems, that was solved by unsetting http(*)_proxy variables, this branch was simply trying to test this behavior https://github.com/grpc/grpc/issues/9987#issuecomment-370156099

Reply:
Okay, I as read the issue, it did not seems related to our problem (even if the name of the error code is the same). It seems that the issue is caused by the setup of their proxy !